bulbs/utils/methods.py

```diff
@@ -99,3 +100,12 @@ def get_overridable_template_name(parent_name, child_name):
         template_name = parent_name
 
     return template_name
+
+
+def get_video_object(videohub_id):
+    video_model = apps.get_model(settings.VIDEO_MODEL)
+    try:
+        video = video_model.objects.get(videohub_ref_id=int(videohub_id))
+    except:
```

Comment:
"Catch-all' dangerous. I would either 
1) catch specific `DoesNotExist` exception
2) rename to `get_video_object_or_404` and raise a 404 exception on error
        ```return get_object_or_404(video_model, videohub_ref_id=videohub_id)```